chain/indexer/Cargo.toml

```diff
@@ -20,14 +20,15 @@ futures = { workspace = true }
 lazy_static = { workspace = true }
 multiaddr = "0.18"
 smart-default = { workspace = true }
+sea-orm = { workspace = true }
```

Comment:
This would be slightly better without the sea-orm and only the hopr-db-api as a dependency.

Reply:
Unfortunately there needs to be lots of partial updates, which would require us to add tons of small functions to the API.

Reply:
But then this sort of leaks the orm objects. What happens, when you wanna use postgres? Such a change would ripple through the entire indexer codebase in the form of different entities... we should think about a way to do this in a micro API fashion that retains some transactionability.

Reply:
We can think about the Visitor pattern. But I'd say let's make it work first.